### ISSUE TITLE
afx/wh csgo crash

### ISSUE THREAD

**Issue by stuqedits** [advancedfx/advancedfx#409]
Hi,
The title says it all, when I execute afx/wh in HLAE, CS:GO crashes automatically.
I could not find a workaround, and here I ask for help.
Thanks in advance.

**Comment by dtugend** [comment]
I could not test myself yet, since the PC is pretty busy with compiling, but it works fine for another HLAE member.
Which CS:GO version are you using (maybe verify cache in Local Files tab of CS:GO Properties (right click in Steam Library))?
Which HLAE version are you using?

**Comment by stuqedits** [comment]
Hi, thanks for responding.
I am using the latest version of CS:GO and the latest version of HLAE, I have no other problems except that (executing afx/wh), which I doubt I have an issue with as the file works fine right next to updateworkaround in "AfxHookSource\assets\csgo\cfg\afx".
I heard 2 rumoured fixes: either disabling the anti-virus, which is not my case as I do not use any, the second being installing HLAE through the installer instead of the zip, I'll give that one a shot.

**Comment by stuqedits** [comment]
After a bit of troubleshooting, I have come to the understanding that the problem lies within the demo, I have tried executing afx/wh in other demos and it works absolutely fine.
Here is the demo link (dust2):
https://www.hltv.org/matches/2299470/natus-vincere-vs-liquid-iem-san-jose-2015
Would you mind testing it?

**Comment by dtugend** [comment]
## Workaround

For old maps / demos only:

Please copy the afx/wh.cfg to your own.cfg file and **remove the following commands from the end**:

```
// Commands to improve entity detection (https://github.com/advancedfx/advancedfx/issues/23):
mirv_cvar_unhide_all
cl_modelfastpath 0
cl_tlucfastpath 0
cl_brushfastpath 0
```

Use the new cfg file for old maps, instead of afx/wh.cfg!


_However the wh will look a bit different (it will mostly affect dead player models color to be white instead of yellow / blue for some)._ 

## Background

This reverses the commands in afx/wh cfg added due to [mirv_streams: e.g. ragdolls not detectable by entity handle #23](https://github.com/advancedfx/advancedfx/issues/23).
Basically the commands expose an normally inaccessible code-path in CS:GO that is normally not used and has bugs that lead to the crash.

**Comment by dtugend** [comment]
I updated the command above, the commands need to be removed actually, sol best is to make own afx/wh.cfg file as said above.

**Comment by stuqedits** [comment]
Thank you!

**Comment by dtugend** [comment]
I hope you don't mind if I reopen the issue: it's not fixed, just a workaround.
